src/App.tsx

```diff
@@ -1,61 +1,117 @@
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import React, { useState } from 'react';
+import { Todo } from './components/Types/Todo';
+import { User } from './components/Types/User';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const getUserById = (userId: number): User | null => (
+  usersFromServer.find(user => user.id === userId) || null
+);
+
+const initialTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.id),
```

Comment:
The `user` property is being assigned using `getUserById(todo.id)`, but it should use `todo.userId` to correctly associate the user with the todo item. Change this to `getUserById(todo.userId)`.